dashboard/test/ui/features/jigsaw.feature

```diff
@@ -11,13 +11,26 @@ Background:
 Scenario: Loading the first jigsaw level
   Then there's an image "jigsaw/blank.png"
 
-Scenario: Can't delete blocks
-  Then the workspace has "1" blocks of type "jigsaw_2A"
-  Then I drag block "1" to offset "-500, 0"
-  # Block wasn't deleted
-  And the workspace has "1" blocks of type "jigsaw_2A"
-  # Make sure block was actually dragged
-  And block "1" is at offset "-176, 20"
+Scenario: Can't delete blocks or lose them outside the workspace
+  Given the workspace has "1" blocks of type "jigsaw_2A"
+  And block "1" is at a location "start_position"
+
+  When I drag block "1" to offset "2000, 0"
+  Then block "1" has not been deleted
+  And block "1" is not at location "start_position"
+  And block "1" is visible in the workspace
+
+  When I drag block "1" to offset "0, 2000"
+  Then block "1" has not been deleted
+  And block "1" is visible in the workspace
+
+  When I drag block "1" to offset "-2000, 0"
+  Then block "1" has not been deleted
+  And block "1" is visible in the workspace
+
+  When I drag block "1" to offset "0, -2000"
+  Then block "1" has not been deleted
+  And block "1" is visible in the workspace
```

Comment:
This is a big change; I tried to be true to the spirit of the original test, and show that the block definitely IS dragged, and that it won't be deleted no matter which way you drag it out of the workspace.